xarray/convert.py

```diff
@@ -53,3 +64,129 @@ def set_cdms2_attrs(var, attrs):
     cdms2_var = cdms2.createVariable(var.values, axes=axes, id=dataarray.name)
     set_cdms2_attrs(cdms2_var, var.attrs)
     return cdms2_var
+
+
+def _pick_attrs(attrs, keys):
+    """ Return attrs with keys in keys list
+    """
+    return dict((k, v) for k, v in attrs.items() if k in keys)
+
+
+def _get_iris_args(attrs):
+    """ Converts the xarray attrs into args that can be passed into Iris
+    """
+    # iris.unit is deprecated in Iris v1.9
+    import cf_units
+    args = {'attributes': _filter_attrs(attrs, iris_forbidden_keys)}
+    args.update(_pick_attrs(attrs, ('standard_name', 'long_name',)))
+    unit_args = _pick_attrs(attrs, ('calendar',))
+    if 'units' in attrs:
+        args['units'] = cf_units.Unit(attrs['units'], **unit_args)
+    return args
+
+
+# TODO: Add converting bounds from xarray to Iris and back
+def to_iris(dataarray):
+    """ Convert a DataArray into a Iris Cube
+    """
+    # Iris not a hard dependency
+    import iris
+    try:
+        from iris.fileformats.netcdf import parse_cell_methods
+    except ImportError:
+        # prior to v1.10
+        from iris.fileformats._pyke_rules.compiled_krb.fc_rules_cf_fc \
+            import _parse_cell_methods as parse_cell_methods
```

Comment:
would it be wroth using `importlib` to clean this up?

Reply:
I wonder if it's worth just removing this catch and requiring iris > 1.10. It's at 1.13 now and nearly 2.0... @pelson?